.github/workflows/MAIN-buildDocker.yml

```diff
@@ -34,31 +34,24 @@ jobs:
           - '16'
           - '17'
         drupal-version:
-          - '10.3.x-dev'
           - '10.4.x-dev'
           - '10.5.x-dev'
-          - '11.0.x-dev'
           - '11.1.x-dev'
+          - '11.2.x-dev'
           - '11.x-dev'
         exclude:
-          - php-version: '8.1'
-            drupal-version: '11.0.x-dev'
-          - php-version: '8.2'
-            drupal-version: '11.0.x-dev'
           - php-version: '8.1'
             drupal-version: '11.1.x-dev'
           - php-version: '8.2'
             drupal-version: '11.1.x-dev'
+          - php-version: '8.1'
+            drupal-version: '11.2.x-dev'
+          - php-version: '8.2'
+            drupal-version: '11.2.x-dev'
           - php-version: '8.1'
             drupal-version: '11.x-dev'
           - php-version: '8.2'
             drupal-version: '11.x-dev'
-          - pgsql-version: '13'
-            drupal-version: '11.0.x-dev'
-          - pgsql-version: '14'
-            drupal-version: '11.0.x-dev'
-          - pgsql-version: '15'
-            drupal-version: '11.0.x-dev'
           - pgsql-version: '13'
```

Comment:
Do we want 11.2 to be run on Postgresql 13, 14, 15 or should it be excluded here too like 11.1 + 11.x?